### DISCUSSION TITLE
Is it possible to have nested Input fields?

### DISCUSSION THREAD

**Discussion by Nosferatu31** [react-dropzone/react-dropzone#1331] [Q&A] [answered]
I have a nested Input field and it does not show the File Dialog window, is it a bug?

**Comment by rolandjitsu** [comment, marked as answer]
@Nosferatu31 does https://react-dropzone.js.org/#section-event-propagation answer your question? It would be helpful if you'd share an example of what you're doing and what the behaviour is.

**Reply by Nosferatu31** [reply]
Hi @rolandjitsu, can't recall how I solved it but I think it was by stopping the event propagation yes.
